module_utils/ansible_nailgun_cement.py

```diff
@@ -186,6 +197,29 @@ def find_lifecycle_environment(module, name, organization, failsafe=False):
     return handle_find_response(module, response, message="No lifecycle environment found for %s" % name, failsafe=failsafe)
 
 
+def find_location(module, name, failsafe=False):
+    loc = Location(name=name)
+    response = loc.search(set(), {'search': 'name="{}"'.format(name)})
+    return handle_find_response(module, response, message="No location found for %s" % name, failsafe=failsafe)
+
+
+def find_operatingsystem(module, name, failsafe=False):
+    response = OperatingSystem(name=name).search(set(), {'search': 'name={}'.format(name)})
+    return handle_find_response(module, response, message="No operating system found for %s" % name, failsafe=failsafe)
+
```

Comment:
Are you writing this different to other entity-search function because Nailgun is inconsistent with Oses?